CorsixTH/Lua/ui.lua

```diff
@@ -1120,6 +1078,58 @@ function UI:afterLoad(old, new)
   if old < 5 then
     self.editing_allowed = true
   end
+  if old < 176 then
+    if self.app.good_install_folder and not self.app.using_demo_files then
+      local gfx = self.app.gfx
+      local palette
+      -- load drug casebook sprite table
+      gfx.cache.palette["DrugN01V.pal"] = nil
```

Comment:
Probably needs to be split up into each dialog

Reply:
I fixed the bug the comments are referring to in this PR, but I don't think the afterload in ui is enough to fix old saves. It wasn't for bank_manager. That's the main reason I don't consider this PR ready yet, and why I added my own comment.